browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -362,6 +362,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 	_cleanText: function(text) {
 		if (!text)
 			return '';
+		if (text.endsWith('...'))
```

Comment:
Depending if IE11 support would still be required, this would need to be adjusted to be backward compatible.

Reply:
IE11 support was dropped. So no need to care.